src/types/api/form.ts

```diff
@@ -18,4 +18,5 @@ export type FormUpdateParams = {
   status?: IForm['status']
   title?: IForm['title']
   webhook?: IForm['webhook']
+  msgSrvcName?: IForm['msgSrvcName']
```

Comment:
not needed if we are not using FormUpdateParams (and we shouldn't since it's deprecated)